vm.go

```diff
@@ -86,10 +88,11 @@ func (exo *Client) doVirtualMachine(action string, params url.Values, async Asyn
 	return r.VirtualMachine, nil
 }
 
-func (exo *Client) GetVirtualMachine(id string) (*VirtualMachine, error) {
+// GetVirtualMachine
+func (exo *Client) GetVirtualMachine(virtualMachineId string) (*VirtualMachine, error) {
```

Comment:
A `ListVirtualMachine` by name would be useful I think too

Reply:
eventually, indeed.